src/components/NavLink/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Link } from '@reach/router';
+import { NavLink as RRDNavLink } from 'react-router-dom';
```

Comment:
Since this component was already named NavLink I had to import the NavLink from react-router-dom as something else to get around compiler errors.